README.md

```
@@ -1,101 +1,91 @@
-### 응답 및 임베딩 비교 코드:
+## 📱 Screenshots
 
-- OpenAI API를 활용한 답변 생성과, 임베딩을 통한 유사도 계산(또는 RAG 등) 로직을 모두 이 파일에 작성합니다.
+<p align="center">
+  <img src="https://github.com/user-attachments/assets/ec89d711-9ec4-4095-b50d-05db3541813d" width="500"/>
+  <img src="https://github.com/user-attachments/assets/ec6178fa-d7e4-455e-9bca-3cdf232e49b5" width="500"/>
+</p>
 
-- 예: generate_response(user_message) 함수 안에서 임베딩 -> 유사도 검색 -> ChatCompletion 호출 과정을 구현.
+<p align="center">
+  <img src="https://github.com/user-attachments/assets/5a0180da-4d0b-4831-9845-fbdc36624706" width="500"/>
```

markdownlint-cli2 (0.20.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)